core/utils/endian.h

```diff
@@ -173,10 +173,6 @@ static_assert(sizeof(be64_t) == 8, "be64_t is not 8 bytes");
 // Copy data from val to *ptr. Set "big_endian" to store in big endian
 bool uint64_to_bin(void *ptr, uint64_t val, size_t size, bool big_endian);
 
-// Copy data from *ptr to *pval. Set "big_endian" if *ptr has big endian data
-bool bin_to_uint64(uint64_t *pval, const void *ptr, size_t size,
-                   bool big_endian);
-
```

Comment:
Just of curiosity, is there any reason to delete this util function? Guessing the functions are used by nowhere now.

Reply:
That was the reason (this was the last user). I wasn't sure if deleting it was appropriate, but nobody objected to the previous three-commit version deleting it. 😺 I can leave it in, or we can always resurrect it from the Git history.